tests/integration-tests/tests/ultraserver/test_gb200/test_gb200/91_nvidia_imex_prolog.sh

```diff
@@ -80,6 +84,47 @@ function get_dna_parameter() {
   jq -r ".cluster.${1}" "${DNA_JSON_FILE}"
 }
 
+function check_imex_needs_reload() {
+  local _expected_ips=$1
+  local _imex_config_file=$2
+  
+  # First check if IMEX service is running
+  if ! systemctl is-active ${IMEX_SERVICE} &>/dev/null; then
+    info "IMEX service is not running, reload needed"
+    return 0  # Need reload
+  fi
+  
+  # Get current IMEX status
+  local imex_status_output
+  if ! imex_status_output=$(timeout 30 /usr/bin/nvidia-imex-ctl -N -j -c "${_imex_config_file}" 2>/dev/null); then
```

Comment:
My hesitation to use this command line is that if the nvidia-imex service goes down or has incoorect configuration, I dont think we can rely on nvidia-imex-ctl.